### DISCUSSION TITLE
de/1-05-ai-models/

### DISCUSSION THREAD

**Discussion by giscus** [cogneon/lernos-ai#47] [General]
# de/1-05-ai-models/

https://ai.lernos.org/de/1-05-ai-models/

**Comment by simondueckert** [comment]
[Meta Llama 3](https://llama.meta.com/llama3/) ist jetzt verfügbar, es gibt 8B und 70B Variante.

**Reply by rednaxelaesor** [reply]
Da hier ja laufend neue Modell zu erwarten sind macht es ggf Sinn, analog zur Grafik am Anfang, bei der Tabelle einen Datumsstand mit anzugeben. 
Eine laufende Aktualisierung macht bei der Geschwindigkeit m. E ~~kaum~~ keinen Sinn.

**Reply by simondueckert** [reply]
Das hatten wir auch diskutiert. Unser Ansatz um den Leitfaden einigermaßen "evergreen" zu halten ist, nur Modelle zu listen, die nachhaltig Bestand haben (GPT, Llama, Gemini) und auch nicht die Versionen zu nennen (also nicht GPT4, sondern nur GPT).

**Comment by KathaZ** [comment]
A1/Z3: sich und (Komma weg)
A1/Z2: ein aus den Trainingsdaten "gelerntes" System => lernendes? bzw. ein System, das aus den Trainingsdaten lernt
A2/Z3: doppeltes Leerzeichen vor Satzanfang "Jede" löschen
A2/Z2: Gesichtszüge*,* Mimik und Beleuchtung
A2/Z6: "Image Generation Model*s*"
A4/Z5: Aufmerksamkeitsmechanismen,
A4/Z9: "Vorhergehende Konzepte basieren Wortvorhersagen ausschließlich auf der Grundlage früherer Wörter." Liest sich schwer. Eher so? => In vorhergehenden Konzepten entstehen Wortvorhersagen ausschließlich auf der Grundlage früherer Wörter.
A4/Z10: Aufmerksamkeits*m*echanismus
A4/Z14: Open-Source-Lizenz
A5/Z7: Parameter *macht* Modelle
A5/Z9: Open-Source-Nutzungslizenz
A5/Z15: Blogbeiträge, Artikel,
Abschnitt "Modelle zur Bilderzeugung"/Z2: Stilübertragung oder (Komma weg)
Abschnitt "Modelle zur Bilderzeugung"/Z3: oder Bewegtbilder*n*
Tabelle "Modelle zur Bilderzeugung"/Open Journey: 
- Hugging Face: Fehlt hier Erklärtext?
- erzeugen und wegen (Komma weg)
Generell: *du, dir, dein* immer klein schreiben
"Deepfakes" verlinken: https://de.wikipedia.org/wiki/Deepfake

**Comment by DJHamburg** [comment]
Man könnte zum Thema offfene versus geschlossene LLMs die aktuellen Infos/Grafik von Maxime Labonne ergänzen: https://www.linkedin.com/posts/maxime-labonne_quick-update-to-this-figure-with-the-addition-activity-7222182483679703042-dzLH?utm_source=share&utm_medium=member_desktop. 
Besser grafisch aufbereitet hat das die FAZ: https://media0.faz.net/ppmedia/w2480/pro/d-economy/kuenstliche-intelligenz/2845727253/1.9886514/original_aspect_ratio_hq/hq/breit-d-24kw31-ki-modelle.jpg.webp (Grafik aus diesem Artikel vom 31.7.2024: https://www.faz.net/pro/d-economy/kuenstliche-intelligenz/modelle-und-akteure-im-ueberblick-so-tickt-die-ki-welt-19887873.html)

**Reply by simondueckert** [reply]
Danke für den Link, den nehme ich mal mit in den Pool für die weiterführenden Links. Grafiken können wir leider nicht einfügen, weil beide nicht unter offenen Lizenze  stehen.
